src/scancode/cli.py

```diff
@@ -131,6 +131,20 @@ def print_about(ctx, param, value):
 
     scancode -f json -l -c samples/zlib/ > scan.json
 
+Scan a directory while ignoring a single file. Print scan results to stdout as JSON:
+
+    scancode --ignore README samples/
+
+Scan a directory while ignoring all files with txt extension. Print scan results to
+stdout as JSON:
```

Comment:
We need some help text that explain which glob characters are allowed in a pattern and how they behave.